snuba/cli/subscriptions_scheduler.py

```diff
@@ -146,6 +150,10 @@ def subscriptions_scheduler(
         storage is not None
     ), f"Entity {entity_name} does not have a writable storage by default."
 
+    logger.info("Checking Clickhouse connections")
+    cluster = storage.get_cluster()
+    check_clickhouse_connections([cluster])
```

Comment:
nit: I'd move this to the line before `build_consumer`, rather than in the middle of validating the CLI args